tests/solvers/python/test_ray_rllib.py

```diff
@@ -0,0 +1,120 @@
+import gymnasium as gym
+import ray.rllib.utils
+from ray.rllib.algorithms.ppo import PPO
+
+from skdecide.core import autocast_all
+from skdecide.domains import Domain
+from skdecide.hub.domain.gym import GymDomain
+from skdecide.hub.domain.rock_paper_scissors import RockPaperScissors
+from skdecide.hub.solver.ray_rllib.ray_rllib import AsRLlibMultiAgentEnv, RayRLlib
+from skdecide.utils import rollout
+
+
+def test_as_rllib_env():
+    domain = RockPaperScissors()
+    env = AsRLlibMultiAgentEnv(domain)
+
+    # check action space
+    assert isinstance(env.action_space, gym.spaces.Dict)
+    assert env.get_agent_ids() == set(env.action_space)
+    for subspace in env.action_space.values():
+        assert isinstance(subspace, gym.spaces.Space)
+
+    # check observation space
+    assert isinstance(env.observation_space, gym.spaces.Dict)
+    assert env.get_agent_ids() == set(env.observation_space)
+    for subspace in env.observation_space.values():
+        assert isinstance(subspace, gym.spaces.Space)
+
+    # checks from ray.rllib
+    ray.rllib.utils.check_env(env)
+
+
+def test_as_rllib_env_with_autocast_from_singleagent_to_multiagents():
+    ENV_NAME = "CartPole-v1"
+
+    domainupcasted = GymDomain(gym.make(ENV_NAME))
+    autocast_all(domainupcasted, GymDomain, RayRLlib.T_domain)
+    env = AsRLlibMultiAgentEnv(domainupcasted)
+
+    # check action space
+    assert isinstance(env.action_space, gym.spaces.Dict)
+    assert env.get_agent_ids() == set(env.action_space)
+    for subspace in env.action_space.values():
+        assert isinstance(subspace, gym.spaces.Space)
+
+    # check observation space
+    assert isinstance(env.observation_space, gym.spaces.Dict)
+    assert env.get_agent_ids() == set(env.observation_space)
+    for subspace in env.observation_space.values():
+        assert isinstance(subspace, gym.spaces.Space)
+
+    # checks from ray.rllib
+    ray.rllib.utils.check_env(env)
+
+
+def test_ray_rllib_solver():
+    # define domain
+    domain_factory = lambda: RockPaperScissors()
+    domain = domain_factory()
+
+    # check compatibility
+    assert RayRLlib.check_domain(domain)
+
+    # solver factory
+    config_factory = lambda: PPO.get_default_config().resources(
```

Comment:
A config_factory seems overkill here (why not just `config`?)

Reply:
I added a comment in the code to explain it.

Reply:
Oh OK, and then I suppose that the config being frozen is an issue (even if we don't modify it?)...